app/hellodjango/locations/helpers/django_admin.py

```
@@ -0,0 +1,94 @@
+    keys_to_test = [
+        tof for tof in target_object_fields if tof in model_keys_and_names.keys()
+    ]
+
+    for k in keys_to_test:
+        foreign_model = model_keys_and_names[k]
+        try:
+            foreign_geom_object = foreign_model.objects.get(
+                geom__intersects=target_object.geom
+            )
+            setattr(target_object, k, foreign_geom_object)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)